src/query/settings/src/settings_default.rs

```diff
@@ -460,6 +460,12 @@ impl DefaultSettings {
                     mode: SettingMode::Both,
                     range: Some(SettingRange::Numeric(0..=100)),
                 }),
+                ("window_partition_spilling_to_disk_bytes_limit", DefaultSettingValue {
+                    value: UserSettingValue::UInt64(10<<30),
```

Comment:
Maybe `value: UserSettingValue::UInt64(1<<30)` ?

Reply:
Why default disable local spill?

Reply:
Because local spill is not yet ready for production, it still needs to be tested in some production environments before we make it available to all users.